### ISSUE TITLE
 %%sql cell magic command raises error

### ISSUE THREAD

**Issue by parthea** [GoogleCloudPlatform/datalab#829]
I'm unable to use the %%sql cell magic command in the latest master. I receive the following error:

```
> %%sql
> SELECT * FROM SampleDataset.SampleTable

NameErrorTraceback (most recent call last)
/usr/local/lib/python2.7/dist-packages/IPython/core/formatters.pyc in __call__(self, obj)
    337                 pass
    338             else:
--> 339                 return printer(obj)
    340             # Finally look for special method names
    341             method = _safe_get_formatter_method(obj, self.print_method)

/usr/local/lib/python2.7/dist-packages/datalab/bigquery/commands/_bigquery.pyc in _repr_html_query_results_table(results)
    958 
    959 def _repr_html_query_results_table(results):
--> 960   return _table_viewer(results)
    961 
    962 

/usr/local/lib/python2.7/dist-packages/datalab/bigquery/commands/_bigquery.pyc in _table_viewer(table, rows_per_page, fields)
    914   if fields is None:
    915     fields = datalab.utils.commands.get_field_list(fields, table.schema)
--> 916   div_id = _html.Html.next_id()
    917   meta_count = ('rows: %d' % table.length) if table.length >= 0 else ''
    918   meta_name = str(table) if table.job is None else ('job: %s' % table.job.id)

NameError: global name '_html' is not defined

```

**Comment by gramster** [comment]
That is fixed in https://github.com/GoogleCloudPlatform/datalab/pull/827.
Just waiting on review.

**Comment by parthea** [comment]
Closing this issue since #827 was merged. I ran into another issue #841
